packages/token-lists/lists/pendulum.json

```
@@ -0,0 +1,48 @@
+{
+    "name": "pendulum",
+    "timestamp": "2024-01-29T00:00:00.000Z",
+    "tokens": [
+      {
+        "networkId": 200,
```

Should be `300` for all of them.
```suggestion
        "networkId": 300,
```